forte/data/readers/ontonotes_reader.py

```diff
@@ -117,6 +126,15 @@ def __init__(self, column_format: Optional[List[Optional[str]]] = None):
             if field not in seen_fields:
                 raise ValueError(f"'{field}' field is required")
 
+    @classmethod
+    def default_configs(cls):
+        config: dict = super().default_configs()
+
+        config.update({
+            "column_format": cls._DEFAULT_FORMAT
```

Comment:
it would be helpful to add the description of the column format as the docstring for this class.

Reply:
Done